engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -581,8 +582,10 @@ class WorkflowActor(workflowToStart: WorkflowToStart,
 
     def deleteFiles() = {
       val rootWorkflowId = data.workflowDescriptor.get.rootWorkflowId
+      val rootWorkflowRoots = data.initializationData.data.values.collect({case Some(i: StandardInitializationData) => i.workflowPaths.workflowRoot}).toSet[Path]
```

Comment:
Mild preference: make this a method on `WorkflowActorData` or `AllBackendInitializationData` to sequester this implementation behind a clearly named function (I don't know the code well enough to say which I prefer)

Reply:
I agree, it's much nicer within `AllBackendInitializationData`.